include/shellstrings.h

```diff
@@ -52,33 +75,43 @@ void shell_save();
  * Returns:
  * 	- None.
  */
- void shell_print(char** lines);
+ void shell_print(char **lines);
 
+/*
+ * shell_input - Input file confirmation prompt for the shell.
+ *
+ * Parameters:
+ *  - input_file: String of the file input by the user.
+ *  - status: String for the file type - dictionary or 
+ *              this is a string because shell_input is used
+ *              to print a confirmation message on the screen
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_input(char *filename, char *status, bool *color);
 
 /*
- * shell_error - Error message printing function for shell, with a string input for specific error
+ * shell_error - Default error prompt string for the shell
  *
  * Parameters:
  *  - error_text: String for the specific error text.
  *
  * Returns:
  *  - None.
  */
-void shell_error(char* error_text);
+void shell_error(char *error_text, bool *color);
 
 /*
- * shell_input - Input file confirmation prompt for the shell.
+ * shell_usage - command line usage instructions
  *
  * Parameters:
- *  - input_file: String of the file input by the user.
- *  - status: String for the file type - dictionary or 
- *				this is a string because shell_input is used
- *				to print a confirmation message on the screen
+ *  - None.
  *
  * Returns:
  *  - None.
  */
-void shell_input(char* input_file, char* status);
+void shell_usage(bool *color);
 
 /*
  * shell_help - Default text input prompt string for the shell.
```

Comment:
Please update this header to accurately reflect the changed function.

Reply:
updated!